apollo-router/tests/docs.rs

```diff
@@ -0,0 +1,33 @@
+#[test]
+#[cfg(target_family = "unix")]
+fn check_config_json() {
+    use std::path::Path;
+
+    use inflector::Inflector;
+    use serde_json::Value;
+
+    // Sanity check consistency and that files exist
+    let config = serde_json::from_str(include_str!("../../docs/source/config.json"))
+        .expect("docs json must be valid");
+    let result = jsonpath_lib::select(&config, "$.sidebar..*").expect("values must be selectable");
+    for value in result {
+        if let Value::String(path) = value {
+            if !path.starts_with("https://") {
+                assert!(
+                    path.replace('/', "").is_kebab_case(),
```

Comment:
not sure this really warrants adding a new dependency

Reply:
It's only in dev? But I could replace with a regex.

Reply:
Annyingly I had auto-merge on, will follow up.